app/views/users/new.html.erb

```diff
@@ -0,0 +1,92 @@
+<!--
+# <form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
- [ ] [OPTIONAL] Create a form partial so you don't have to repeat the code on both the user new and show views.